src/quick-lint-js/fe/parse.h

```diff
@@ -398,7 +398,8 @@ class parser {
 
   template <class ExpectedParenthesesError, class ExpectedParenthesisError,
             bool CheckForSketchyConditions, bool CheckForCommaOperator>
-  void parse_and_visit_parenthesized_expression(parse_visitor_base &v);
+  void parse_and_visit_parenthesized_expression(source_code_span token,
+                                                parse_visitor_base &v);
```

Comment:
The `parse_visitor_base &v` parameter should come first.
```suggestion
  void parse_and_visit_parenthesized_expression(parse_visitor_base &v,
                                                source_code_span token);
```